services/fncsgossbridge/service/fncs_goss_bridge.py

```diff
@@ -44,11 +44,13 @@
 @author: fish334
 @author: poorva1209
 """
+import argparse
 import cmath
 from datetime import datetime
 import json
 import math
 import os
+from Queue import Queue
```

Comment:
I believe this should be lowercase from queue import Queue at least it is with py3

Reply:
oooh this may be an 2.7/3.x issue?